build/bootstrap.proj

```diff
@@ -17,7 +17,6 @@
         <PackageDownload Include="Microsoft.VisualStudio.Composition" version="[15.8.98]" />
         <PackageDownload Include="Microsoft.VisualStudio.ProjectSystem" version="[16.0.201-pre-g7d366164d0]" />
         <PackageDownload Include="Microsoft.Web.Xdt" version="[2.1.2]" />
-        <PackageDownload Include="Newtonsoft.Json" version="[9.0.1]" />
```

Comment:
This used to be done for signing purposes....can you can check the assembly that ends up in the vsix is signed?

Reply:
I didn't test a build before pushing the previous commit, and there was no `Newtonsoft.Json` going into the VSIX. I didn't want to spend time to investigate how to get the dll from a `PackageReference`, so I just updated the version number here and in the vsix project.

Reply:
nvm, I thought this removed this reference :D 
Disregard this.